charts/argo-cd/values.yaml

```diff
@@ -1752,6 +1654,258 @@ repoServer:
 
 ## Argo Configs
 configs:
+  # General Argo CD configuration
+  ## Ref: https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/argocd-cm.yaml
+  # TODO: Should we add more examples from link above / keep example for oidc.config?
```

Comment:
Yes, why not removing unrelevant settings. The less examples, the less work needed to have a look into when there's a new release coming in.